cli/command/formatter/disk_usage.go

```diff
@@ -144,57 +177,91 @@ func (ctx *DiskUsageContext) verboseWrite() error {
 			}
 		}
 
-		err := ctx.contextFormat(tmpl, &imageContext{
+		duc.Images = append(duc.Images, &imageContext{
 			repo:  repo,
 			tag:   tag,
-			trunc: true,
+			trunc: ctx.trunc,
```

Comment:
Do you really need this `ctx.trunc` ?
It's not possible to just locally to this function declare a variable trunc?
```go
trunc := ctx.Context.Format.IsTable()
```

Reply:
Why? You just need the `ctx` to know if it needs to be truncated or not? 
```
ctx.Context.Format.IsTable()
```
Or maybe I'm misreading something?

Reply:
@silvin-lubecki you are correct, a previous version of my code needed this. I removed the field.